Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1898,13 +1908,22 @@ void CGContextSetPatternPhase(CGContextRef context, CGSize phase) {
 #pragma endregion
 
 #pragma region Drawing Operations - Text
+
+// Attributes are needed in DrawGlyphRuns but names are defined in CoreText
```

Comment:
worried about these getting out of sync, would possibly prefer defining here and letting CoreText = these